addons/website/data/image_library.xml

```diff
@@ -709,5 +709,11 @@
     <field name="type">url</field>
     <field name="url">/website/static/src/img/snippets_demo/s_popup.jpg</field>
 </record>
+<record id="website.s_closer_look_default_image" model="ir.attachment">
+    <field name="public" eval="True"/>
+    <field name="name">s_closer_look_default_image.jpg</field>
+    <field name="type">url</field>
+    <field name="url">/website/static/src/img/snippets_demo/s_closer_look.jpg</field>
```

Comment:
The image looks very blurred 🥲
Also, it would be nice to remove the arrow on the top-left.

Since the device takes a huge portion of the screen, what do you think about creating a "full screen" picture with a browser window? eg
![image](https://github.com/user-attachments/assets/e985176a-83bb-471a-a2fa-ea2222c9454d)

Reply:
@stefanorigano can you check now ? I made the same change in https://github.com/odoo/odoo/pull/175902
Thanks !